src/language/source.js

```diff
@@ -41,3 +41,15 @@ export class Source {
     );
   }
 }
+
+Object.defineProperty(Source, Symbol.toStringTag, {
```

Comment:
Why add this to the constructor itself? Shouldn't only instances be encountered? Also, wouldn't this produce confusing results? I would expect a constructor function to toString as a function